### ISSUE TITLE
New update broke JP verification

### ISSUE THREAD

**Issue by E-pyx** [FlashChaser/Open-Source-Battle-Bot#120]
As stated, the newest update for JP broke the verification process and you can't update either side due to command prompt crashing.

**Comment by tasinttttttt** [comment]
Confirmed also for global

**Comment by LBK1557** [comment]
It works for Global, just broken for JP.

**Comment by hungryjimmy** [comment]
Yeah, Global seems to be working fine, JP verification process broken.

**Comment by Thorax96** [comment]
its not the verfication process. They switched from http to https and also seem to have changed a little bit in the general infrastructure. For anyone wanting to try it out youself, just replace the http with https and try debugging it.

**Comment by LBK1557** [comment]
> its not the verfication process. They switched from http to https and also seem to have changed a little bit in the general infrastructure. For anyone wanting to try it out youself, just replace the http with https and try debugging it.

I'll try that, thanks.

**Comment by Thorax96** [comment]
Update : its the client version which is apparently formatted wrong.  If I find out what changes need to be done I will create a pull request

**Comment by hann12** [comment]
I tried it several times but it doesn't start ...
Will it be available in the next update?
I hope it will be usable again.

**Comment by shiroyasha888** [comment]
No new fix?

**Comment by E-pyx** [comment]
If the apparent fix is so easy, can we get an updated release with said fix. It's been quite a while since the issue started.

**Comment by Thorax96** [comment]
> If the apparent fix is so easy, can we get an updated release with said fix. It's been quite a while since the issue started.

Who said it is easy? But if you can't wait either use Karys Bot or contribute on fixing this one.

**Comment by hann12** [comment]
KarysBot also doesn't seem to work JP..

**Comment by hungryjimmy** [comment]
You sure? I used it perfectly fine yesterday to complete the clash battlefield.

**Comment by hann12** [comment]
KarysBot??
I couldn't even do the first DL.

**Comment by hungryjimmy** [comment]
Yeah, just got the EXE from the discord server.
https://discord.gg/pVuNyde

**Comment by hann12** [comment]
Thank you!

**Comment by FedorRampage** [comment]
Any update on the JP farming bot?

**Comment by hungryjimmy** [comment]
Did you even read the issue thread? Click on the discord link I sent a bit earlier and you can download a fully functional bot from the server.

**Comment by FedorRampage** [comment]
@hungryjimmy , i know about Kary's bot. Since the recent update kary's bot is not working on JP. (Pc only and linux are working). i'm using my Mac or Android to farm.

**Comment by hungryjimmy** [comment]
@FedorRampage my bad. The only quick fix I can think of is running some kind of virtual machine or Windows emulator to run it, but other than that I'm unaware of solutions, sorry.

**Comment by Cptspiller** [comment]
@hungryjimmy I tried Karys bot and 3 seconds into it the thing crashed and my save was lost.. and Bandai wont recover it so yeah.. 960ish day account gone... not happy at all.

**Comment by FedorRampage** [comment]
Did you try it on Global or JP?

**Comment by Cptspiller** [comment]
Global ios... bandai said they wont recover because the account couldn't be verified.. never had this problem with this bot.. so wouldn't recommenced karys.. i'm mad and disappointed now as i spent way too much time and money on that account.

**Comment by Cptspiller** [comment]
Hoping this bot gets fixed or they make it where I can select to only update global database as I don't play JP. This bot was perfect.

**Comment by FedorRampage** [comment]
I used the bot on Global last week on Global iOS, the bot was working fine!
Sorry to hear about your account...

**Comment by hann12** [comment]
Will there be any updates in the future?
This was easy to use.
I am really sad.

**Comment by FedorRampage** [comment]
No update as far  As I know.

**Comment by hann12** [comment]
Is that so...
